test/spec/runner/Operations.zig

```diff
@@ -0,0 +1,235 @@
+                tc.post = try BeaconStateAllForks.init(fork, post_state);
+            }
+            return tc;
+        }
+
+        pub fn deinit(self: *Self) void {
+            if (comptime @hasDecl(OpType, "deinit")) {
+                OpType.deinit(self.pre.allocator, &self.op);
+            }
+            self.pre.deinit();
+            if (valid) {
+                self.post.deinit(self.pre.allocator);
+            }
+        }
+
+        pub fn process(self: *Self) !void {
+            const verify = switch (self.bls_setting) {
+                .default, .required => true,
+                .ignored => false,
+            };
+
+            switch (operation) {
+                .attestation => {
+                    const attestations_fork: ForkSeq = if (fork.gte(.electra)) .electra else .phase0;
+                    var attestations = @field(ssz, attestations_fork.forkName()).Attestations.default_value;
+                    defer attestations.deinit(self.pre.allocator);
+                    const attestation: *@field(ssz, attestations_fork.forkName()).Attestation.Type = @ptrCast(@alignCast(&self.op));
+                    try attestations.append(self.pre.allocator, attestation.*);
+                    const atts = attestations;
+                    const attestations_wrapper: state_transition.Attestations = if (fork.gte(.electra))
+                        .{ .electra = &atts }
+                    else
+                        .{ .phase0 = &atts };
+
+                    try state_transition.processAttestations(self.pre.allocator, self.pre.cached_state, attestations_wrapper, verify);
+                },
+                .attester_slashing => {
+                    try state_transition.processAttesterSlashing(OpType.Type, self.pre.cached_state, &self.op, verify);
+                },
+                .block_header => {
+                    return error.SkipZigTest;
+                    // TODO: processBlockHeader currently takes signed block which is incorrect. Wait for it to accept unsigned block.
+                    // try state_transition.processBlockHeader(self.pre.allocator, self.pre.cached_state, &self.op);
+                },
+                .bls_to_execution_change => {
+                    try state_transition.processBlsToExecutionChange(self.pre.cached_state, &self.op);
+                },
+                .consolidation_request => {
+                    try state_transition.processConsolidationRequest(self.pre.allocator, self.pre.cached_state, &self.op);
+                },
+                .deposit => {
+                    try state_transition.processDeposit(self.pre.allocator, self.pre.cached_state, &self.op);
+                },
+                .deposit_request => {
+                    try state_transition.processDepositRequest(self.pre.allocator, self.pre.cached_state, &self.op);
+                },
+                .execution_payload => {
+                    return error.SkipZigTest;
```

Comment:
I think we can call `processExecutionPayload`. What's the reason of skipping it?